cmd/cvc-operator/controller/cstorvolumeclaim.go

```diff
@@ -512,6 +519,19 @@ func getUsablePoolListForClone(pvName, srcPVName string, poolList *apis.CStorPoo
 	return usablePoolList
 }
 
+// prioritizedPoolList prioritized pool instance name matched to the given
+// nodeName in case of replica affinity is enabled via volume policy
+func prioritizedPoolList(nodeName string, list *apis.CStorPoolInstanceList) *apis.CStorPoolInstanceList {
+	for i, pool := range list.Items {
+		if pool.Spec.HostName != nodeName {
```

Comment:
Here, we are checking nodeName with HostName. Will it work if nodeName differs from hostName?

We are populating nodeName
https://github.com/openebs/maya/blob/5d066c6fd24f0dbe9a463096929b83f741731a5b/pkg/cstor/poolinstance/v1alpha3/build.go#L192

Reply:
We have to fix in CSPI spec

Reply:
this need to be fixed

Reply:
how this got fixed?